packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -153,7 +153,7 @@ export class PendingStateManager implements IDisposable {
 				message.sequenceNumber !== undefined,
 				0x97c /* saved op should already have a sequence number */,
 			);
-			return message.sequenceNumber >= (snapshotSequenceNumber ?? 0);
+			return message.sequenceNumber > (snapshotSequenceNumber ?? 0);
```

Comment:
What a simple fix, was this all that there was to fix?

Reply:
Looks like that's the case but it worries me about our current tests not catching some scenarios like this one. Seems like I lot of test coverage is still missing.